InventoryManagementSystem/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ConnectionStrings": {
+    "SqlDB": "Data Source=TASBEH-TAKRORE;Initial Catalog=SimpleInventory;Integrated Security=True",
```

Comment:
make sure that you're not adding the connection string to a remote DB that could be accessed from another machine.